libs/wingc/src/lsp/completions.rs

```diff
@@ -857,11 +931,24 @@ fn get_completions_from_namespace(
 			}
 		}
 	}
+
+	let is_wing_file = matches!(namespace.module_path, ResolveSource::WingFile);
+
 	namespace
 		.envs
 		.iter()
 		.flat_map(|env| env.symbol_map.iter())
 		.flat_map(|(name, symbol)| format_symbol_kind_as_completion(name, &symbol.kind))
+		// ignore the builtin-functions
+		.filter(|c| {
```

Comment:
I'm wondering if we should filter out all non-public symbols when searching a namespace for completions

Reply:
Good call, that's much better